project/dagger-query-ui-example.sh

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+bazel run src/com/google/daggerquery/example:beach_example
```

Comment:
For the example, I think we should only need to use "bazel build" in this command.

Reply:
Actually `//src/com/google/daggerquery/example:beach_example` is already specified [within](https://github.com/googleinterns/dagger-query/blob/efea4d441fd6534ad482ed2138058c1c766f07b1/project/src/com/google/daggerquery/BUILD#L29) `//src/com/google/daggerquery:example_java_server`, so I think bazel should figure out the dependency graph, and no need for manually building `src/com/google/daggerquery/example:beach_example` separately beforehand?

Reply:
O, thanks, you are right - I have forgotten that we construct binding graph at compile-time and running is not required. I have removed the first command.